node/network/collator-protocol/src/validator_side/mod.rs

```diff
@@ -979,6 +979,8 @@ where
 	Sender: CollatorProtocolSenderTrait,
 	I: IntoIterator<Item = ((ParaId, Hash), Vec<BlockedAdvertisement>)>,
 {
+	state.metrics.time_request_unblocked_collations();
```

Comment:
Does this actually observe anything? I believe it's dropped immediately.

`let _timer = state.metrics.time_request_unblocked_collations();`